iphone/Classes/TiUIClipboardProxy.m

```diff
@@ -65,18 +65,110 @@ static ClipboardType mimeTypeToDataType(NSString *mimeType)
 
 @implementation TiUIClipboardProxy
 
+-(id) init
+{
+    if (self = [super init])
+    {
+        shouldCreatePasteboard = true;
+        persistent = false;
+        isNamedPasteBoard = false;
+        isUnique = false;
```

Comment:
We need these variables as pasteboard will be created on combination of these. If we create  pasteboard using [self pasteboard] , all calculation will happen inside pasteboard function and every time we use pasteboard (e.g read name ) [self pasteboard] will get called and all calculation will happen again. So I think current implementation is fine.